packages/react-native/Libraries/StyleSheet/processBackgroundImage.js

```diff
@@ -123,25 +140,30 @@ function parseCSSLinearGradient(
   while ((match = linearGradientRegex.exec(cssString))) {
     const gradientContent = match[1];
     const parts = gradientContent.split(',');
-    let points = TO_BOTTOM_START_END_POINTS;
+    let orientation: LinearGradientOrientation = DEFAULT_ORIENTATION;
     const trimmedDirection = parts[0].trim().toLowerCase();
     const colorStopRegex =
       /\s*((?:(?:rgba?|hsla?)\s*\([^)]+\))|#[0-9a-fA-F]+|[a-zA-Z]+)(?:\s+(-?[0-9.]+%?)(?:\s+(-?[0-9.]+%?))?)?\s*/gi;
```

Comment:
Can we add a comment explaining what this regex matches to and same with the linearGradientRegex?

 eg from processFilter:   // matches on functions with args and nested functions like "drop-shadow(10 10 10 rgba(0, 0, 0, 1))"

Or maybe a link to the syntax we are matching from the spec?